src/snowflake/snowpark/_internal/compiler/cte_utils.py

```diff
@@ -0,0 +1,133 @@
+#
+# Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
```

Comment:
will we have two cte_utils.py files now? Why not move everything into this one file?

Reply:
Oh, yes, i intended to remove it, that is a mistake, i removed this file